src/app/in-memory-data.service.ts

```diff
@@ -0,0 +1,32 @@
+import { Injectable } from '@angular/core';
+import { InMemoryDbService } from 'angular-in-memory-web-api';
+import { Hero } from './hero';
```

Comment:
Codacy has a fix for the issue: [Strings must use doublequote.](https://app.codacy.com/gh/infinity-arc/Kobus-tour-of-heroes/pullRequest?prid=7838883)
```suggestion
import { Hero } from "./hero";
```